src/test/components/MenuButtons.test.js

```diff
@@ -98,89 +107,52 @@ describe('app/MenuButtons', function() {
       (shortenUrl: any).mockImplementation(() =>
         Promise.resolve('https://profiler.firefox.com/')
       );
+      // Node does not have URL.createObjectURL.
+      (URL: any).createObjectURL = () => 'mockCreateObjectUrl';
     });
 
-    it('matches the snapshot', () => {
+    it('matches the snapshot for the closed state', () => {
       const { container } = setup();
-      expect(Array.from(container.children)).toMatchSnapshot();
+      expect(container).toMatchSnapshot();
```

Comment:
I introduced a bunch of snapshots here. It'd be better to have a more targeted approach, I filed #1896 as a follow-up. I didn't want to over-invest as this area is likely to change as we nail down the user flow.